king_phisher/client/windows/campaign_import.py

```diff
@@ -238,17 +238,41 @@ def select_xml_campaign(self):
 			campaign_xml = ET.parse(target_file)
 		except ET.ParseError as error:
 			self.logger.error("cannot import campaign: {0} is not a valid XML file".format(target_file), error)
-			raise KingPhisherInputValidationError("{0} is not a valid xml file".format(target_file))
+			gui_utilities.show_dialog_error(
+				'Improper Format',
+				self.window,
+				'{} is not a valid XML file'.format(target_file)
```

Comment:
I'd prefer to just drop the target_file so the message will always be the same length and can be a proper sentence.